tests/integration/docusaurus/connecting_to_your_data/how_to_configure_an_inferredassetdataconnector.py

```diff
@@ -583,12 +583,9 @@
 ] = "../data/nested_directories_data_asset/"
 
 test_python = context.test_yaml_config(
-    yaml.dump(datasource_config), return_mode="report_object"
+    yaml.dump(datasource_config),
 )
 
-# NOTE: The following code is only for testing and can be ignored by users.
-assert test_yaml == test_python
```

Comment:
Removing this assertion as `report_object` returned a dict (which allowed for equality checks). It now returns a `Datasource` (which I believe uses memory addresses for __eq__)